### PR TITLE
New dashboard/timeseries

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
Adding zero data values to time series. Adds utils test.

Screen Shot 2020-07-28 at 1 55 01 pm

_Versus_

Screen Shot 2020-07-28 at 3 15 51 pm

**Todo**

- ~~[ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request~~
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/server/api/credit_transfer_api.py [hunk 1]
```diff
@@ -178,6 +178,10 @@ def post(self, credit_transfer_id):
         target_balance = post_data.get('target_balance')
 
         transfer_use = post_data.get('transfer_use')
+        try:
+            use_ids = transfer_use.split(',')  # passed as '3,4' etc.
+        except AttributeError:
+            use_ids = transfer_use
```
Comment (tristanhcole) [inline app/server/api/credit_transfer_api.py:185]: Confused as to where/how this is being used?

Comment (enjeyw) [reply app/server/api/credit_transfer_api.py:185]: ohhh i can comment here!
this was originally buried inside the create payment method - the mobile passes back transfer usages as a string. Ideally, we wouldn't have to parse strings at all, but in cases where we do, this process should be at the api level not buried in a method

Comment (tristanhcole) [reply app/server/api/credit_transfer_api.py]: just force pushed

Comment (tristanhcole) [reply app/server/api/credit_transfer_api.py]: I think these changes are supposed to be on #394

Comment (michielderoos) [reply app/server/api/credit_transfer_api.py]: Ahh, Sorry!

Comment (tristanhcole) [reply app/server/api/credit_transfer_api.py]: All good!

**Pull request merged**